lib/tasks/ovirt.rake

```diff
@@ -0,0 +1,65 @@
+namespace :ovirt do
+  desc <<~END_DESC
+    Remove all oVirt / RHEV compute resources & profiles from Foreman.
+    All hosts associated with these compute resources will be disassociated and kept.
+    All host groups with oVirt compute resources & profiles will have their compute resource and compute profile removed.
+
+    Examples:
+      # foreman-rake ovirt:drop
+  END_DESC
+
+  task :drop => :environment do
+    compute_resources = ComputeResource.unscoped.where(type: 'Foreman::Model::Ovirt')
+
+    nothing_to_do if compute_resources.empty?
+
+    ask_user
```

Comment:
If we're going to call this from automated scripts (foreman-maintain comes to mind) we need a way to disable this prompt.

@evgeni would you agree and if so, what's a good way to do this?

Reply:
how about env variables, something like `SKIP_OVIRT_CHECK`

Reply:
I have no idea how we did it in the past (e.g. Puppet), so probably not the right person to ask, but env var doesn't sound crazy

Reply:
I did think about an env var, but something like `ASSUME_YES`. The more specific inclusion of ovirt is IMHO not a bad thing, but I wouldn't call this a check. Perhaps `OVIRT_ASSUME_YES` as a compromise?
```suggestion
    ask_user unless ENV['OVIRT_ASSUME_YES'] == '1'
```

Possibly also reuse some string to bool conversion instead of only checking for `'1'`